addons/l10n_ar_withholding/models/account_move.py

```diff
@@ -0,0 +1,12 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models
+
+
+class AccountMove(models.Model):
+
+    _inherit = 'account.move'
+
+    withholding_ids = fields.One2many(
```

Comment:
it has to be on account.move?  (as it only applies to payments...)

Reply:
check, and if not, at least add the prefix

Reply:
Done!